Sources/Rendering/Core/Picker/index.d.ts

```diff
@@ -21,39 +21,34 @@ export interface vtkPicker extends vtkAbstractPicker {
 	getActors(): vtkActor[];
 
 	/**
-	 * Get a pointer to the dataset that was picked (if any).
+	 * Get the dataset that was picked (if any).
 	 */
 	getDataSet(): any;
 
 	/**
 	 * Get mapper that was picked (if any)
 	 */
-	getMapper(): null | vtkMapper;
+	getMapper(): Nullable<vtkMapper>;
 
 	/**
 	 * Get position in mapper (i.e., non-transformed) coordinates of pick point.
 	 */
-	getMapperPosition(): number[];
+	getMapperPosition(): Vector3;
 
 	/**
-	 * 
+	 * Get a list of the points the actors returned by getActors were intersected at.
 	 */
-	getMapperPositionByReference(): number[];
```

Comment:
This methods should not be removed
![Capture d’écran du 2024-01-13 10-44-04](https://github.com/Kitware/vtk-js/assets/878518/459a8f3e-fb01-4f63-9405-ef3604f9d122)